sklearn/tests/test_common.py

```diff
@@ -83,16 +85,20 @@ def test_all_estimators():
 
 def test_non_meta_estimators():
     # input validation etc for non-meta estimators
-    # FIXME these should be done also for non-mixin estimators!
-    estimators = all_estimators(type_filter=['classifier', 'regressor',
-                                             'transformer', 'cluster'])
+    estimators = all_estimators()
     for name, Estimator in estimators:
+        if issubclass(Estimator, BiclusterMixin):
+            continue
+        if name.endswith("HMM") or name.startswith("_"):
+            continue
+        if name not in CROSS_DECOMPOSITION + ['SelectFdr']:
```

Comment:
I submitted a PR for this. In the mean time why not increase `alpha` to a large enough value?